src/UI/Internal/Primitives.elm

```diff
@@ -1,14 +1,29 @@
-module UI.Internal.Primitives exposing (roundedBorders, roundedFields)
+module UI.Internal.Primitives exposing (relativeRoundedBorders, roundedBorders)
 
 import Element exposing (Attribute)
 import Element.Border as Border
+import UI.Internal.Size as Size exposing (Size)
 
 
 roundedBorders : Attribute msg
 roundedBorders =
     Border.rounded 8
 
 
-roundedFields : Attribute msg
-roundedFields =
-    Border.rounded 8
+{-| REVIEW: I don't think I liked this name, gimme suggestions for this!
+-}
+relativeRoundedBorders : Size -> Attribute msg
```

Comment:
- `roundedBordersFrom : Size -> Attribute msg`
- `roundedBorders : Size -> Attribute msg` and call the other one as `defaultRoundedBorders`

Not sure tho 🤔